Sources/Html/Node.swift

```diff
@@ -16,6 +19,29 @@ public enum Node {
   case text(String)
 }
 
+extension Node {
+  public var isEmpty: Bool {
+    switch self {
+    case let .comment(string), let .doctype(string), let .raw(string), let .text(string):
+      return string.isEmpty
+    case .element:
+      return false
+    case let .fragment(children):
+      return children.isEmpty || children.allSatisfy { $0.isEmpty }
+    }
+  }
+}
+
+public prefix func ... (nodes: [Node]) -> Node {
+  return .fragment(nodes.filter { !$0.isEmpty })
```

Comment:
Not sure I understand why this filter is needed. What's up with it?

Reply:
Was just trying to get rid of the nesting and slipped it in here. Probably not necessary or probably better as an explicit traversal?